feat(web-hosting): add How to access your Web Hosting migrated from D… (#6729)
scaleway/docs-content: pages/webhosting/how-to/access-migrated-webhosting.mdx

The migration is transparent: your websites stay online with no downtime, your cPanel, FTP, and database credentials remain the same. 
After the migration, you can only manage your Web Hosting from the Scaleway console.

This page explains how to access your Web Hosting after the migration, and what changes for you.

<Message type="important">
    - From July 28, 2026, the Online console is no longer available to manage your Web Hosting. You must log in to the [Scaleway console](https://account.scaleway.com/login). 
    - The migration only applies to Web Hosting. Any domain names or other products you hold with Online remain on the Online console.
</Message>

<Requirements />

- A Dedibox Web Hosting Cloud (cPanel) plan (Essential, Performance, or Premium) migrated to Scaleway on July 28, 2026
- Your Online account login credentials, which remain unchanged after the migration

## How to access your Web Hosting after migration from Dedibox to Scaleway

1. Go to the [Scaleway console login page](https://account.scaleway.com/login).
2. Log in using your Online credentials.

> 1. Click **Web Hosting** in the **Domains & Web Hosting** section of the [console](https://console.scaleway.com/) side menu. The list of your Web Hosting plans appears.

Author: ```suggestion
3. Click **Web Hosting** in the **Domains & Web Hosting** section of the [console](https://console.scaleway.com/) side menu. The list of your Web Hosting plans appears.
```

4. Click the Web Hosting plan you want to manage.
The **Web Hosting overview** displays, where you can manage your hosting, cPanel, FTP accounts, databases, and email accounts.

## About the migration

### What changes

- Only your Web Hosting plan is migrated to Scaleway.
- Your hosting runs on a more modern infrastructure, giving you access to the latest PHP versions.

### What stays the same

- Your plan, price, and specifications (disk space, number of mailboxes, and other limits) remain unchanged.
- Domain names and any other products you hold with Online are not migrated and continue to be managed on the Online console.

### Billing and plan management

- Billing and plan management (such as termination) remain on the [Online console](https://console.online.net/en/login) for now.

> - Plan management is planned to move to the Scaleway console at a later date. Scaleway will notify you before this change. Until then, you cannot modify your migrated plan from the Scaleway console. 

Reviewer: ```suggestion
- Plan management is scheduled to move to the Scaleway console at a later date. Scaleway will notify you before this change. Until then, you cannot modify your migrated plan from the Scaleway console. 
```

Reviewer: This is just so we don't have the word "plan" twice so close to each other.

---
title: How to access your Web Hosting migrated from Dedibox to Scaleway
description: Learn how to access and manage your Dedibox cPanel Web Hosting after its automatic migration to the Scaleway console
tags: webhosting migration cpanel dedibox online
dates:
  validation: 2026-07-24
---
import Requirements from '@macros/iam/requirements.mdx'

> As part of the integration of Dedibox Web Hosting into the Scaleway ecosystem, all Dedibox Web Hosting Cloud plans powered by cPanel are automatically migrated to Scaleway Web Hosting on 28 July 2026. This migration applies to the three cPanel plans: Essential, Performance, and Premium.

Reviewer 2: ```suggestion
As part of the integration of Dedibox Web Hosting into the Scaleway ecosystem, all Dedibox Web Hosting Cloud plans powered by cPanel are automatically migrated to Scaleway Web Hosting on July 28, 2026. This migration applies to the three cPanel plans: Essential, Performance, and Premium.
```

The migration is transparent: your websites stay online with no downtime, your cPanel, FTP, and database credentials remain the same. 
After the migration, you can only manage your Web Hosting from the Scaleway console.

This page explains how to access your Web Hosting after the migration, and what changes for you.

<Message type="important">

>     - From 28 July 2026, the Online console is no longer available to manage your Web Hosting. To access and manage your hosting, you must log in to the [Scaleway console](https://account.scaleway.com/login). 

Reviewer 2: ```suggestion
    - From July 28, 2026, the Online console is no longer available to manage your Web Hosting. To access and manage your hosting, you must log in to the [Scaleway console](https://account.scaleway.com/login). 
```

Reviewer 2: J'ajouterais que "To access and manage your hosting," me paraît superflus et pourrait être enlevé, même je sais que certains sont davantage attachés à des approches didactiques dans l'équipe ^^ En l'occurrence, on est dans une admonition, donc je suis d'avis de faire en sorte que le contenu soit le plus bref possible.

>     - From 28 July 2026, the Online console is no longer available to manage your Web Hosting. To access and manage your hosting, you must log in to the [Scaleway console](https://account.scaleway.com/login). 

Author: ```suggestion
    - From July 28, 2026, the Online console is no longer available to manage your Web Hosting. You must log in to the [Scaleway console](https://account.scaleway.com/login). 
```

>     - The migration only applies to Web Hosting. Any domain names or other products you hold with Online remain on the Online console and are not migrated.

Reviewer 2: "and are not migrated" superflus ? À moins que le propos est de bien souligner qu'on trouve l'information au même endroit, alors dans ce cas j'ajouterais un "only" : "... or other products you hold with Online remain on the Online console only." par exemple, mais à affiner

>     - The migration only applies to Web Hosting. Any domain names or other products you hold with Online remain on the Online console and are not migrated.

Author: ```suggestion
    - The migration only applies to Web Hosting. Any domain names or other products you hold with Online remain on the Online console.
```

</Message>

<Requirements />

> - A Dedibox Web Hosting Cloud (cPanel) plan (Essential, Performance, or Premium) migrated to Scaleway on 28 July 2026

Reviewer 2: ```suggestion
- A Dedibox Web Hosting Cloud (cPanel) plan (Essential, Performance, or Premium) migrated to Scaleway on July 28, 2026
```

## How to access your Web Hosting after migration from Dedibox to Scaleway

1. Go to the [Scaleway console login page](https://account.scaleway.com/login).
2. Log in using your Online credentials.
1. Click **Web Hosting** in the **Domains & Web Hosting** section of the [console](https://console.scaleway.com/) side menu. The list of your Web Hosting plans appears.
4. Click the Web Hosting plan you want to manage.
The **Web Hosting overview** displays, where you can manage your hosting, cPanel, FTP accounts, databases, and email accounts.

## About the migration

### What changes

- Only your Web Hosting plan is migrated to Scaleway.
- Your hosting runs on a more modern infrastructure, giving you access to the latest PHP versions.

### What stays the same

- Your plan, price, and specifications (disk space, number of mailboxes, and other limits) remain unchanged.
- Domain names and any other products you hold with Online are not migrated and continue to be managed on the Online console.

### Billing and plan management

- Billing and plan management (such as termination) remain on the [Online console](https://console.online.net/en/login) for now.

> - Plan management is planned to move to the Scaleway console at a later date. Scaleway notifies you before this change.

Reviewer 2: Je suis vraiment pour utiliser le présent aussi souvent que possible, mais ici, "Scaleway notifies you" me paraît très étrange d'autant plus qu'on a "at a later date" juste au-dessus.

> - Plan management is planned to move to the Scaleway console at a later date. Scaleway notifies you before this change.

Author: ```suggestion
- Plan management is planned to move to the Scaleway console at a later date. Scaleway will notify you before this change.
```

> - Until then, you cannot modify your migrated plan from the Scaleway console. 

Reviewer 2: Pourquoi est-ce un bullet séparé ? Ça m'a l'air d'être une information liée au précédent. Je suis d'ailleurs assez incertaine sur l'usage de bullet points dans cette section qui a une tonalité davantage explicative et conversationnelle que les précédentes.

> - Until then, you cannot modify your migrated plan from the Scaleway console. 

Author: ```suggestion
- Plan management is planned to move to the Scaleway console at a later date. Scaleway will notify you before this change. Until then, you cannot modify your migrated plan from the Scaleway console. 
```
